app/Main.hs

```diff
@@ -92,7 +92,7 @@ main = do
   localEnvVars <- getEnvironment
   cliAndEnvOptions <- parseOptionsFromEnvAndCli localEnvVars
 
-  if oDebug cliAndEnvOptions
+  if (oLogLevel cliAndEnvOptions) >= Info
```

Comment:
My intuition with log level "info or higher" would be, that it's "info, warning, error, critical".
But that would not make sense here, since you want to print this if it's "info or debug", right?

In general "higher" log level, ususally means "more serious". E.g. in Python as well: https://docs.python.org/2/library/logging.html#logging-levels

See also my comment about the `Ord` instance below.